packages/api-demo/src/app.json

```diff
@@ -4,7 +4,8 @@
         "name": "SSF Desktop Wrapper",
         "url": "http://localhost:8080/index.html",
         "uuid": "ssf-desktop-api-openfin-demo",
-        "autoShow": true
+        "autoShow": true,
+        "preload": "http://localhost:8080/scripts/containerjs-api-bundle.js"
```

Comment:
Yeah possibly, I hadn't thought of that. It still works the same as before like this, but you're right in that the `bundle.js` wouldn't be in the html anymore so we wouldn't need to check for `fin` being defined or anything, making the bundle redundant.